### ISSUE TITLE
Better output `propertylookup` and `offsetlookup`

### ISSUE THREAD

**Issue by evilebottnawi** [prettier/plugin-php#373]
Input:
```php
$SomeVeryLongUpperCaseConstant->someVeryLongCallExpression()->some_very_long_member_expression;
$weNeedToReachTheEightyCharacterLimitXXXXXXXXXXXXXXXXX->someNode
->childrenInAnArray[0];
$superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered;
$superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered[0];
```
Output:
```php
$SomeVeryLongUpperCaseConstant->someVeryLongCallExpression()->
    some_very_long_member_expression;
$weNeedToReachTheEightyCharacterLimitXXXXXXXXXXXXXXXXX->someNode->
    childrenInAnArray[0];
$superSupersuperSupersuperSupersuperSupersuperSuperLong->
    exampleOfOrderOfGetterAndSetterReordered;
$superSupersuperSupersuperSupersuperSupersuperSuperLong->
    exampleOfOrderOfGetterAndSetterReordered[0];
```

Also need add tests for `offsetlookup`

**Comment by mgrip** [comment]
@evilebottnawi was this fixed with #372?

**Comment by evilebottnawi** [comment]
@mgrip need add tests and look :smile:
